x-pack/plugins/ml/public/alerting/jobs_health_rule/tests_selection_control.tsx

```diff
@@ -0,0 +1,121 @@
+  onChange: (update: JobsHealthRuleTestsConfig) => void;
+  errors?: string[];
+}
+
+export const TestsSelectionControl: FC<TestsSelectionControlProps> = ({ config, onChange }) => {
+  const uiConfig = getResultJobsHealthRuleConfig(config);
+
+  const updateCallback = useCallback(
+    (update: Partial<Exclude<JobsHealthRuleTestsConfig, undefined>>) => {
+      onChange({
+        ...(config ?? {}),
+        ...update,
+      });
+    },
+    [onChange, config]
+  );
+
+  return (
+    <EuiFormFieldset
+      legend={{
+        children: i18n.translate(
+          'xpack.ml.alertTypes.jobsHealthAlertingRule.testsSelection.legend',
+          {
+            defaultMessage: 'Select health checks to perform',
+          }
+        ),
+      }}
+    >
+      <EuiSwitch
+        label={
+          <FormattedMessage
+            id="xpack.ml.alertTypes.jobsHealthAlertingRule.testsSelection.datafeedCheck.label"
+            defaultMessage="Datafeed is not started"
+          />
+        }
+        onChange={updateCallback.bind(null, { datafeed: { enabled: !uiConfig.datafeed.enabled } })}
+        checked={uiConfig.datafeed.enabled}
+      />
+
+      <EuiSpacer size="s" />
+
+      {false && (
+        <>
+          <EuiSwitch
+            label={
+              <FormattedMessage
+                id="xpack.ml.alertTypes.jobsHealthAlertingRule.testsSelection.mmlCheck.label"
+                defaultMessage="Model memory limit monitoring"
+              />
+            }
+            onChange={updateCallback.bind(null, { mml: { enabled: !uiConfig.mml.enabled } })}
+            checked={uiConfig.mml.enabled}
+          />
+
+          <EuiSpacer size="s" />
+
+          <EuiSwitch
+            label={
+              <FormattedMessage
+                id="xpack.ml.alertTypes.jobsHealthAlertingRule.testsSelection.delayedDataCheck.label"
+                defaultMessage="Delayed data"
+              />
+            }
+            onChange={updateCallback.bind(null, {
+              delayedData: { enabled: !uiConfig.delayedData.enabled },
+            })}
+            checked={uiConfig.delayedData.enabled}
+          />
+
+          <EuiSpacer size="s" />
+
+          <EuiSwitch
+            label={
+              <FormattedMessage
+                id="xpack.ml.alertTypes.jobsHealthAlertingRule.testsSelection.jobBehindRealtimeCheck.label"
+                defaultMessage="Job is running behind real-time"
+              />
+            }
+            onChange={updateCallback.bind(null, {
+              behindRealtime: { enabled: !uiConfig.behindRealtime.enabled },
+            })}
+            checked={uiConfig.behindRealtime.enabled}
+          />
+
+          <EuiSpacer size="s" />
+
+          <EuiSwitch
+            label={
+              <FormattedMessage
+                id="xpack.ml.alertTypes.jobsHealthAlertingRule.testsSelection.errorMessagesCheck.label"
+                defaultMessage="There are errors in job messages"
```

Comment:
```suggestion
                defaultMessage="There are errors in the job messages."
```

Reply:
Fixed in https://github.com/elastic/kibana/pull/106084/commits/9c0b3ecfa61edfea97ca0881482ca69af5ebd954